build/Common.props

```diff
@@ -57,7 +57,7 @@
     <PackageReference Include="StyleCop.Analyzers" Version="$(StyleCopAnalyzersPkgVer)" Condition="'$(SkipAnalysis)'!='true'">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.CodeCoverage" Version="$(MicrosoftCodeCoveragePkgVer)" Condition="'$(Configuration)'=='Release'"/>
+    <PackageReference Include="Microsoft.CodeCoverage" Version="$(MicrosoftCodeCoveragePkgVer)" Condition="'$(Configuration)'=='Debug'"/>
```

Comment:
the CI build+test runs in Release right? so it wont have codecoverage. Won't `PrivateAssets` help here?

Reply:
You are right, that also solves my problem. Updated.

Reply:
I would keep the release configuration.

Reply:
I would also rename the PR :)

Reply:
Renamed. Why keep the release configuration? Does this slow down the build?

Reply:
The idea is to not download things you won't use.
Since this is only used during release, we did this way to download only when needed.

Reply:
I added it back, although I would say this is probably premature optimisation...

Reply:
> The idea is to not download things you won't use.
> Since this is only used during release, we did this way to download only when needed.

I disagree.
I think the principle here is we want DEBUG and RELEASE to be as close as possible, so the DEBUG version actually makes sense to exist (if they are so different, I would challenge us that the DEBUG version is not useful at all).